lib/fika/parser/non_literal_exps.ex

```diff
@@ -49,6 +49,7 @@ defmodule Fika.Parser.NonLiteralExps do
     ])
     |> label("function call")
 
+  # TO-DO: accept nested if-else expressions
```

Comment:
@emilsoman During tests, I noticed that if-else doesn't allow an if-else expression inside itself. I'd rather deal with this in a separate PR

Reply:
Sure!